pandas/core/groupby.py

```diff
@@ -2368,6 +2368,13 @@ def _get_grouper(obj, key=None, axis=0, level=None, sort=True,
     # axis of the object
     if level is not None:
         if not isinstance(group_axis, MultiIndex):
+            if isinstance(level, list) or isinstance(level, range):
+                if len(level) > 1:
```

Comment:
i don't think checking the length is not valid. Following works in MI.

```
pd.DataFrame([[1, 1], [2, 2], [3, 3]], index=pd.MultiIndex.from_arrays([[1, 1, 1], [1, 2, 3]])).groupby(level=[0, 0]).sum()
```

 I think it should raise if level is not scalar.